substratest/factory.py

```diff
@@ -15,21 +15,26 @@
 
 DEFAULT_SUBSTRATOOLS_VERSION = '0.5.0'
 
-# TODO improve opener get_X/get_y methods
-# TODO improve metrics score method
-
-DEFAULT_OPENER_SCRIPT = """
+DEFAULT_OPENER_SCRIPT = f"""
+import csv
 import json
+import os
 import substratools as tools
 class TestOpener(tools.Opener):
     def get_X(self, folders):
-        return folders
+        res = []
+        for folder in folders:
+            with open(os.path.join(folder, '{DEFAULT_DATA_SAMPLE_FILENAME}'), 'r') as f:
+                reader = csv.reader(f)
+                for row in reader:
+                    res.append(int(row[0]))
+        return res
```

Comment:
Just a note: `get_X` should return a matrix (not an array like I said in my previous comment!). But we're good here because the returned array can be considered as a matrix that only has 1 column.